hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -66,7 +66,7 @@
 public class TcpIpConnectionManager implements ConnectionManager, Consumer<Packet> {
 
     private static final int RETRY_NUMBER = 5;
-    private static final int DELAY_FACTOR = 100;
+    private static final long DELAY_FACTOR = 100L;
```

Comment:
May I ask why is this change desired?

Reply:
Sure, based on [this sonar bug](https://sonarcloud.io/project/issues?id=hz-master%3Amaster&open=AWLd3b6T-pl6AHs2TnVt&resolved=false&severities=MINOR&types=BUG), however, it is currently unlikely but since there is multiplication, I added it.

Reply:
Also, we have many sonar minor bug analysis similar to this one, based on decision on this PR, I will either solve them as `wont fix` or send similar updates.

Reply:
IMHO I would leave it as it is, because it reduces the memory footprint. But maybe we can reconsider, WDYT @mmedenjak ?

Reply:
In this case it isn't an issue. Constants don't have a significant impact on memory footprint since it will just take an additional 32 bytes per JVM (per classloader actually).